django_restql/fields.py

```diff
@@ -268,7 +268,7 @@ def to_internal_value(self, data):
                 )
             elif data == empty and required:
                 data = default
-            elif data == empty and default == empty:
+            elif data == empty and default == empty and not self.is_partial:
```

Comment:
The problem is not here, even if we do this those sending PATCH request would still experience a bug if they put nothing in a nested field, the bug would cause a nested field value to be replaced with null.

Reply:
I'm not sure I understand, I didn't write unit tests, but I did test this change manually.  If you patch and omit the nested field, it is unaffected (which is precisely what you'd want on a patch operation that doesn't mention a given field, you leave it alone).  If you patch specifically with null in the JSON, then it will set it to None in the python, and therefore null in the DB.

Reply:
Check tests added on #145